.github/workflows/build_and_test.yml

```diff
@@ -68,7 +68,12 @@ jobs:
           pip install "git+https://github.com/mage-ai/singer-python.git#egg=singer-python"
       - name: Run unit tests
         run: |
-          python3 -m unittest discover -s mage_ai --failfast
+          python3 -m unittest discover -s mage_ai --failfast -p "^(?!.*test_ai_function).*$"
+      - name: Run AI unit tests
```

Comment:
No need to add this. You can rebase master. We exclude 3.7 in our unit tests.

Reply:
Sounds good! Reverted the build_and_test yaml change.